tool/plugin/lib/edit.dart

```diff
@@ -136,6 +136,18 @@ List<EditCommand> editCommands = [
     replacement: 'urlClassLoader.addFiles(paths)',
     versions: ['2021.1'],
   ),
+  Subst(
```

Comment:
@stevemessick is this the best way to do this?
The alternative I can think of is to use reflection to dynamically call the api.

Reply:
This allows us to avoid reflection.